public/app/features/migrate-to-cloud/api/endpoints.gen.ts

```diff
@@ -8,7 +8,7 @@ const injectedRtkApi = api.injectEndpoints({
       query: (queryArg) => ({
         url: `/cloudmigration/migration`,
         method: 'POST',
-        params: { authToken: queryArg.authToken },
+        body: queryArg.cloudMigrationRequestBody,
```

Comment:
This is the impact of this fix - the auth token is correctly sent in the post body instead of query string params.